doc/interfaces/remote.rst

```diff
@@ -0,0 +1,94 @@
+Remote
+======
+
+The remote interface works as a networked bridge between the computer running
+the application and the computer owning the physical CAN interface.
+
+Server
+------
+
+The computer which owns the CAN interface must start a server which accepts
+incoming connections. If more than one channel is to be shared, multiple
+servers must be started on different ports.
+
```

Comment:
Will do. Since we create one bus per connection, it might be backend dependent if it will allow multiple connections.